extensions/optaplanner/runtime/src/main/java/io/quarkus/optaplanner/OptaPlannerObjectMapperCustomizer.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.optaplanner;
+
+import javax.inject.Singleton;
+
+import org.optaplanner.persistence.jackson.api.OptaPlannerJacksonModule;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.quarkus.jackson.ObjectMapperCustomizer;
+
+@Singleton
+public class OptaPlannerObjectMapperCustomizer implements ObjectMapperCustomizer {
```

Comment:
IIRC, this isn't picked up automatically because it's part of the module and therefore if the user doesn't have Jackson you don't register the bean. 
Perhaps this warrants a comment on this class as to how things have been done and why because I am sure we will forget about it in the future

Reply:
Adding this javadoc
```
/**
 * Only registered if the Jackson capability is active.
 * OptaPlanner doesn't use Jackson, but it does have optional Jackson support for {@link Score}, etc.
 */
```

Reply:
Fixed with new commit on this fork.